### ISSUE TITLE
keystore need to be saved under the ETH address (not by meteor.userId)

### ISSUE THREAD

**Issue by jellegerbrandy** [Paratii-Video/paratii-player#181]
Now, keystores are set in localstorage with the following keys:

`keystore-anonymous`
`keystore-{userId}`

the last one needs to become

`keystore-{ethereumadderss}`, so we can find it (in theory) also when we remove the meteor users from the equation

Not sure exactly of all the consequences...

**Comment by eliawk** [comment]
yes but we need to save the ET address also to the user in meteor db to have a connection between the user and keystore

**Comment by jellegerbrandy** [comment]
we already save the ETH account in meteor, in `profile.ptiAddress`.

At the moment, we save extrea user data in 'profile.ptiAddress' in the users collection - which is the collection defined by meteor to manage users.

I proposse to do some refactoring and keeping the meteor-specific things separate from the paratii things, and define a new collection (perhaps call it `accounts`), whose `id` is the Ethereum address of the user. and where we store the info we need.

**Comment by jellegerbrandy** [comment]
Directly related to #248
